docker/mwdb.ini

```diff
@@ -6,3 +6,5 @@ uploads_folder = /app/uploads
 serve_web = 0
 local_plugins_folder = /app/plugins
 local_plugins_autodiscover = 1
+# json or standard
+logging_type = json
```

Comment:
I would remove it from `docker/mwdb.ini`, it should be opt-in.

But it would be nice if you add commented out option to the template with appropriate documentation (`mwdb/templates/mwdb.ini.tmpl`)